data/transaction.go

```diff
@@ -64,6 +64,7 @@ type FullTransaction struct {
 	HyperblockHash                    string                                `json:"hyperblockHash,omitempty"`
 	Receipt                           *transaction.ReceiptApi               `json:"receipt,omitempty"`
 	ScResults                         []*transaction.ApiSmartContractResult `json:"smartContractResults,omitempty"`
+	Logs                              *transaction.LogsAPI                  `json:"logs,omitempty"`
```

Comment:
We have `ReceiptApi`, `ApiSmartContractResult` and `LogsAPI` . can we use a single pattern?

Reply:
done.